### PR TITLE
fix bn tuning issue with CK batchnorm solvers

### PR DESCRIPTION
**Pull request by bghimireamd**
~~* enabled CK solver~~
* Fixed tuning issue with CK batch norm
* batch norm driver iter default to 10 (similar to conv). Before it was 1.

### CODE REVIEW

File: src/include/miopen/solver/implicitgemm_ck_util.hpp [hunk 1]
```diff
@@ -313,11 +313,10 @@ ConvSolution InitAnyInvokerFactory(const ProblemDescriptionType& problem,
...
                 auto invoker_ptr     = sh_conv_ptr->MakeInvokerPointer();
 
-                const auto enable_profiling = handle.IsProfilingEnabled();
-                float elapsed_time =
-                    invoker_ptr->Run(argument_ptr.get(), {handle.GetStream(), enable_profiling});
-                if(enable_profiling)
+                invoker_ptr->Run(argument_ptr.get(), {handle.GetStream(), false});
+                if(handle.IsProfilingEnabled())
                 {
+                    float elapsed_time = handle.GetKernelTime();
```
Comment (BrianHarrisonAMD) [inline src/include/miopen/solver/implicitgemm_ck_util.hpp]: Since we are disabling the timing from CK we will need to track the kernel time ourself.

I think we need to add something like:

```
{
           WorkAroundHipEventProfiler prf(handle);
           invoker_ptr->Run(argument_ptr.get(), {handle.GetStream(), false});
}

**Review by BrianHarrisonAMD — APPROVED** [review]
LGTM
